qermit/noise_model/transpiler_backend.py

```diff
@@ -70,6 +84,17 @@ def run_circuit(
         handle = self.process_circuit(circuit, n_shots, **kwargs)
         return self.get_result(handle=handle)
 
+    def process_circuits(
+        self,
+        circuits: Sequence[Circuit],
+        n_shots: Sequence[int],
+    ) -> List[uuid.UUID]:
```

Comment:
Ah, do you use `uuid.UUID` instead of  `ResultHandle` in this mock backend?

Reply:
Never mind, figured it out :P